### PR TITLE
Frogger game was completed

### PR DESCRIPTION
**Pull request by LGeorge007**
### Object-oriented-JS (Frogger game)

[Demo]https://lgeorge007.github.io/object-oriented-JS/
[Code base]https://github.com/LGeorge007/object-oriented-JS/tree/master

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by LGeorge007** [comment]
When I creating an array of enemies, I forgot to pass the value of the minSpeed ​​variable into the speed parameter. There is now a value of 25, which was when debugging.

File: submissions/LGeorge007/object-oriented-JS/app.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    };
+
+    if (this.y < 0 ){
+        alert("You win!");
+        this.setStartPosition();
+    };
+};
+
+const player = new Player(xStartPositionPlayer, yStartPositionPlayer);
+const allEnemies = [new Enemy(0,280,25,player), new Enemy(0,200,25,player), new Enemy(0,130,25,player)];
```
Comment (OleksiyRudenko) [inline submissions/LGeorge007/object-oriented-JS/app.js:117]: There is a pattern here. Imagine we need dozens of enemies.
In every `new Enemy(0,XXX,25,player)` "XXX" is the only thing that is unique.
Consider the following example:
`[1,22,333].map(y => new Circle(0, y, 100)` creates 3 identical circles at different positions along `y` axis

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@LGeorge007 well done!
Let's make code drier employing `Array` methods.

File: submissions/LGeorge007/object-oriented-JS/app.js (hunk 1 above)
Comment (LGeorge007) [reply submissions/LGeorge007/object-oriented-JS/app.js]: improved.

**Comment by OleksiyRudenko** [comment]
Don't forget to click **Re-request review** when you are ready
image

Otherwise it is not clear if you're still working or ready for the next review iteration.

I just jumped in here occasionally.

**Review by OleksiyRudenko — APPROVED** [review]
@LGeorge007 great job!

**Pull request merged**